### PR TITLE
(PRE-117) fix CI failure

### PR DESCRIPTION
**Pull request by phongdly**
CI failure due to the missing of ubuntu config file as below error logs:
https://jenkins-modules.puppetlabs.com/view/2.%20linux%20only/view/catalog_preview/view/master/job/platform-lang_puppetlabs-catalog-preview_intn-sys_full-pe-2016.2.x-master/PLATFORM=ubuntu1604-64mdca,WORKER_LABEL=beaker/1/console

This PR is simply adding the required config file

### CODE REVIEW

File: spec/integration/nodesets/pe/ubuntu1604-64mdca.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+HOSTS:
+  ubuntu1604-64-1:
+    pe_dir: 
+    pe_ver: 
+    pe_upgrade_dir: 
+    pe_upgrade_ver:
```
Comment (johnduarte) [inline spec/integration/nodesets/pe/ubuntu1604-64mdca.yaml]: nit pick: there are trailing spaces in the four lines above.

**Comment by seangriff** [comment]
The new file looks good.  

Should there be a corresponding file in the parent directory: spec/integration/nodesets ?   (There is a sles12 and sles11 file in the nodesets/pe/ directory and not in nodesets/, so not unprecedented.)

File: spec/integration/nodesets/pe/ubuntu1604-64mdca.yaml (hunk 1 above)
Comment (er0ck) [inline spec/integration/nodesets/pe/ubuntu1604-64mdca.yaml]: beaker-hostgenerator does that 😐

**Comment by er0ck** [comment]
no need to have it in the parent dir.  that's for foss tests, where we don't currently test against ubuntu1604

**Pull request merged**